guidelines/enhancement_template.md

```diff
@@ -74,6 +75,7 @@ around the enhancement process.
 ## Release Signoff Checklist
```

Comment:
As part of this change, we can also remove this entire section. We should make sure the items listed here are represented in the rest of the template, but we don't want people to feel obligated to come back and update the checklist and we won't merge enhancements until all of the necessary parts are filled in.

Reply:
that makes sense to me, the checklist always ends up falling behind.

Reply:
I think the User Docs is the one missing, I see 2 places we can add it template:
1)  between test plan and graduation criteria as its own section to highlight or within graduation criteria
2) as a question in Graduation Criteria

Something like: Does this enhancement require an update to user-facing documentation in openshift-docs?

Reply:
Let's start with a question in the graduation criteria for now. We're going to review that section in a later update and we can decide then if we want to leave the question there or add a new section to talk about doc updates.